tests/dummy/app/index.html

```diff
@@ -14,6 +14,16 @@
     <link rel="stylesheet" href="assets/vendor.css">
     <link rel="stylesheet" href="assets/dummy.css">
 
+    <script type="text/javascript"
+        src="//cdnjs.cloudflare.com/ajax/libs/mathjax/2.6.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
+    </script>
+    <script type="text/x-mathjax-config">
```

Comment:
That's an excellent question. There are two levels on which we can answer. Especially wrt bower, the likely solution now is a mix of these two answers.

- This addon has never been great to setup from scratch, so the install docs don't have much precedent. Even a README section to start would help.
- For a more realistic solution: the addon should handle the installation of external dependencies for the user. (rather than making them read prose and paste in code) This means giving them the option, and making all the steps clear and traceable:
  - Try to install mathjax via bower if possible. Hard-coding the cdn url means that mathjax is hidden from all the tools we would normally use to find and update our dependencies. If someone finds a security vulnerability- we would not know it affected us.
    - Addons are able to tell a consuming app that they (may) need a bower dependency, [during the addon install process](https://ember-cli.com/api/classes/Blueprint.html#method_addBowerPackageToProject) (using hooks in the default blueprint). The consuming app author could then decide whether to accept or remove that bower.json entry. 
  - Some addons provide configuration options that can control whether certain features are used. For the `script` block, ideally the addon would provide the configuration it needs when the app loads. Not immediately sure of the best mechanism for doing this (maybe an initializer of some kind), but see, eg [ember-bootstrap docs](http://www.ember-bootstrap.com/#/getting-started/assets) for inspiration on how this might work.